internal/serve/httphandler/forgot_password_handler.go

```diff
@@ -55,11 +56,38 @@ func (h ForgotPasswordHandler) ServeHTTP(w http.ResponseWriter, r *http.Request)
 	var forgotPasswordRequest ForgotPasswordRequest
 	err = json.NewDecoder(r.Body).Decode(&forgotPasswordRequest)
 	if err != nil {
-		httperror.BadRequest("invalid request body", err, nil).Render(w)
+		httperror.BadRequest("Invalid request body", err, nil).Render(w)
 		return
 	}
 
-	if !h.ReCAPTCHADisabled {
+	// validate request
+	v := validators.NewValidator()
+
+	v.Check(forgotPasswordRequest.Email != "", "email", "email is required")
+
+	if v.HasErrors() {
+		httperror.BadRequest("Request invalid", err, v.Errors).Render(w)
+		return
+	}
+
+	user, err := h.AuthManager.GetUserByEmail(ctx, forgotPasswordRequest.Email)
```

Comment:
💡  For a better UX, it'd be good to sanitize the email field by stripping any extra space and lowercasing it.